ftplugin/latex-suite/bibtools.py

```diff
@@ -171,15 +193,17 @@ def __init__(self, filelist=''):
                 self.addfile(f)
 
     def addfile(self, file):
-        fields = urllib.urlopen(file).read().split('@')
+        fields = urlopen('file://' + quote(os.path.abspath(file))).read().split(b'@')
         for f in fields:
+            f = f.decode('utf-8')
```

Comment:
No, this is a good but annoying question. Apparently it's not entirely clear what encodings bibtex will and won't support. I figured just make it support utf-8: it should still work with ASCII. http://wiki.lyx.org/BibTeX/Tips